src/electionguard_cli/steps/shared/election_builder_step.py

```diff
@@ -0,0 +1,53 @@
+from electionguard.elgamal import ElGamalPublicKey
+from electionguard.group import ElementModQ
+from electionguard.key_ceremony import ElectionJointKey
+from electionguard.election_builder import ElectionBuilder
+from electionguard.utils import get_optional
+from electionguard_cli.cli_models.e2e.e2e_inputs import CliElectionInputsBase
+
+from electionguard_cli.cli_models import BuildElectionResults
```

Comment:
Relative imports

Reply:
That would go up three and producing linting error, see prior comment